source/getting-started/installation-raspberry-pi-image.markdown

```diff
@@ -12,7 +12,7 @@ footer: true
 The easiest way to install Home Assistant on your Raspberry Pi is by using Hassbian: a Raspberry Pi image with Home Assistant built-in. The image will install the latest version of Home Assistant on initial boot (~5 minutes).
 
  1. [Download the latest image][image-download]
- 2. Flash the image to an SD card:
+ 2. Flash the image to a Micro SD card:
```

Comment:
At least the first version of a Raspberry Pi is using an standard-sized SD card.